public/app/features/alerting/unified/components/AlertLabelDropdown.tsx

```diff
@@ -5,6 +5,8 @@ import { GroupBase, OptionsOrGroups, createFilter } from 'react-select';
 import { SelectableValue } from '@grafana/data';
 import { Field, Select, useStyles2 } from '@grafana/ui';
 
+import { t } from '../../../../core/internationalization';
```

Comment:
A nitpick, but not sure how we want to handle it in general: most of the other imports for this in alerting are `app/core/internationalisation` - should we prefer that?

Reply:
my slight preference is actually for the `../../../` style since it requires 0 additional tooling. the aliases always need some extra config in typescript/webpack/whatever else to work which can cause headaches when migrating to a new tool. and the only downside is ugly path, but i always just cmd-click through anyway 🤷